TGC/events/Germany VS Hungary Events.txt

```diff
@@ -0,0 +1,672 @@
+	picture = "german_rhine"
+
+	is_triggered_only = yes
+
+	option = {
+               name = "EVTOPTA2000000002"
+                  diplomatic_influence = {
+	             who = BAN
+	             value = 400
+        
+                  }
+BAN = { country_event = { id = 2000000003 days = 10 } }
+}
+}
+##Banat and Siebenbürgen Government set up##
+ 
+country_event = {
+	id = 2000000003
+	title = "EVTNAME2000000003"
+	desc = "EVTDESC2000000003"
+	picture = "german_rhine"
+
+	is_triggered_only = yes
+
+	option = {
+               name = "EVTOPTA2000000003"
+               government = dominion
+               primary_culture = south_german
+               add_accepted_culture = romanian
+               add_accepted_culture = north_german                         
+               remove_accepted_culture = hungarian
+        }
+
+}
+
+
+##Freistaat Siebenbürgen introduction##
+
+country_event = {
+	id = 2000000004
+	title = "EVTNAME2000000004"
+	desc = "EVTDESC2000000004"
+	picture = "german_rhine"
+
+	is_triggered_only = yes
+
+	option = {
+               name = "EVTOPTA2000000002"
+                  diplomatic_influence = {
+	             who = SIE
+	             value = 400
+                  }
+SIE = { country_event = { id = 2000000003 days = 10 } }
+
+}
+
+}
+##Hungarian client kingdom set up##
+
+country_event = {
+	id = 2000000005
+	title = "EVTNAME2000000005"
+	desc = "EVTDESC2000000005"
+	picture = "german_rhine"
+
+	is_triggered_only = yes
+
+	option = {
+               name = "EVTOPTA2000000005"
+                  diplomatic_influence = {
+	             who = HUN
+	             value = 400
+       
+                  }
+               HUN = { country_event = { id = 2000000006 days = 10 } }
+        }
+}
+
+##Habsburg King arrives##
+
+country_event = {
+	id = 2000000006
+	title = "EVTNAME2000000006"
+	desc = "EVTDESC2000000006"
+	picture = "german_rhine"
+
+	is_triggered_only = yes
+
+	option = {
+               name = "EVTOPTA2000000006"
+               add_accepted_culture = south_german 
+               political_reform = no_voting
+               government = absolute_monarchy
+               set_country_flag = pol_reform_blocker
+                        
+               
+        }
+
+}
+
+
+##Hungarian Ultimatum##
+
+country_event = {
+	id = 2000000007
+	title = "EVTNAME2000000007"
+	desc = "EVTDESC2000000007"
+	picture = "german_rhine"
+
+	is_triggered_only = yes
+
+	option = {
+               name = "EVTOPTA2000000007"
+               HUN = { country_event = { id = 2000000008 days = 5 } }
+               
+               ai_chance = {factor = 10}
+        }       
+        option = {
+               name = "EVTOPTB2000000007"
+               HUN = { country_event = { id = 2000000009 days = 5 } }
+               
+               ai_chance = {factor = 5}
+        }
+        option = {
+               name = "EVTOPTC2000000007"
+               HUN = { country_event = { id = 2000000010 days = 5 } }
+               

```

Comment:
Missing ai_chance on two of these - not sure what behavior is expected in such situations.

Reply:
i think default is that all options are equal and the factor multiplys the chance for that 
i added factor 1 just in case now